go.mod

```diff
@@ -14,7 +14,9 @@
 // module pocket
 module github.com/pokt-network/poktroll
 
-go 1.20
+go 1.21
```

Comment:
Can we revert this? This can have downstream effects, so I'd want to make it a separate PR.